tools/ovc/automation/create_package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import os
+from shutil import rmtree
+
+from utils import Automation
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--build_number", type=str, help="Build number to be added to package version", default="0", )
+args = parser.parse_args()
+
+auto = Automation()
+base_dir = os.path.dirname(__file__)
+bom_path = os.path.join(base_dir, "package_BOM.txt")
```

Comment:
why do we need this file? who is an user of `create_package.py`?

Reply:
Looks like there are no usages. I removed this file and moved package_BOM.txt to unit_tests.